src/test/resources/net/sf/jabref/importer/fileformat/MsBibImporterTest2.bib

```diff
@@ -0,0 +1,45 @@
+% This file was created with JabRef 2.10.
```

Comment:
Where does this bib come from?

It is shown as new, but the header says JabRef 2.10. Where id you have it from? And where is the bibliography data from?

I thought, you were creating these test files for yourself.